### PR TITLE
New nuclear operative item, The Macho Wrestler Bundle!

### PR DESCRIPTION
**Pull request by Therealdoooc213**
## About The Pull Request

Adds a new nuclear operative item, The Macho Wrestler bundle! (18 TC, trademark pending) It contains a Rudos mask (modifies speech), a wrestling belt (grants martial arts), a adrenalin implant, and combat hypospray for healing! Strike fear into your enemys when you throw the captain at a wall and stomp on their head!

## Why It's Good For The Game

Nukies have a severe lack of fun items or flavor, which restricts their gameplay to mainly be Fake centcom, war, or stealth. If there are more nukie items it encourages them to try new gimmicks, plans, and think!

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

https://github.com/user-attachments/assets/231d2adc-b328-49d1-a42c-c9fa4dad978a

</details>

## Changelog
:cl:
add: A new nuclear operative bundle, The Macho Wrestler! Contains everything you need to wrestle the captain into giving you the disk.
/:cl:

### CODE REVIEW

**Comment by ClownMoff** [comment]
Good luck convincing mementainers to add the belt and I mean it honestly. I would like seeing this on the game, Also please add icons to the abilities.

File: code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -264,6 +264,14 @@ GLOBAL_LIST_INIT(illegal_tech_blacklist, typecacheof(list(
 	cost = 30
 	purchasable_from = UPLINK_NUKE_OPS
 
+/datum/uplink_item/bundles_TC/machoman
+	name = "The Macho Wrestler Bundle"
+	desc = "For the STRONGEST operatives! Master your wrastling abilitys with this bundle. \
```
Comment (ClownMoff) [inline code/modules/uplink/uplink_items.dm]: Wrastling 🗿 

Also, abilities*

File: code/modules/uplink/uplink_items.dm [hunk 2]
```diff
@@ -264,6 +264,14 @@ GLOBAL_LIST_INIT(illegal_tech_blacklist, typecacheof(list(
...
+			Contains: A wrestling belt, for only the strongest, a adrenalin implanter, a mask to strike fear into your enemys, and a combat hypospray!"
```
Comment (ClownMoff) [inline code/modules/uplink/uplink_items.dm]: Contains: A wrestling belt, only for the strongest machos, an adrenaline implanter, a luchador mask to strike fear into your enemies, and a combat hypospray!"

**Review by ClownMoff — CHANGES_REQUESTED** [review]
Some typos and bad grammar

File: code/modules/uplink/uplink_items.dm [hunk 3]
```diff
@@ -264,6 +264,14 @@ GLOBAL_LIST_INIT(illegal_tech_blacklist, typecacheof(list(
 	cost = 30
 	purchasable_from = UPLINK_NUKE_OPS
 
+/datum/uplink_item/bundles_TC/machoman
+	name = "The Macho Wrestler Bundle"
+	desc = "For the STRONGEST operatives! Master your wrastling abilities with this bundle. \
```
Comment (Rukofamicom) [inline code/modules/uplink/uplink_items.dm]: I understand what you're going for but Wrastling just looks wrong. Spell it right or find a different way to spell the funny way to say it. `Wrasslin'` has my vote.

**Review by Rukofamicom — CHANGES_REQUESTED** [review]

**Review by Rukofamicom — APPROVED** [review]

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**